src/main/scala/netty/ActorChannelConnector.scala

```diff
@@ -27,12 +41,36 @@ final private class ActorChannelConnector(clients: ActorRef[Clients.Control])(us
             clients ! Clients.Control.Stop(client)
           }
 
-  private def emitToChannel(channel: Channel): ClientEmit =
+  private def intSetting(key: String) =
+    settings.makeSetting(key, config.getInt(key))
+
+  private def emitToChannel(channel: Channel, withFlush: Boolean): ClientEmit =
     case ipc.ClientIn.Disconnect(reason) =>
       channel
         .writeAndFlush(CloseWebSocketFrame(WebSocketCloseStatus(4010, reason)))
         .addListener(ChannelFutureListener.CLOSE)
     case ipc.ClientIn.RoundPingFrameNoFlush =>
       channel.write { PingWebSocketFrame(Unpooled.copyLong(System.currentTimeMillis())) }
-    case in =>
+    case in if withFlush || interval.get <= 0 =>
       channel.writeAndFlush(TextWebSocketFrame(in.write))
+    case in =>
+      channel.write(TextWebSocketFrame(in.write))
+      flushQ.add(channel)
+
+  private def flush(): Unit =
+    val maxDelayFactor  = maxDelay.get.toDouble / interval.get
```

Comment:
Is this on purpose?  Looks like maxDelayFactor resolves to `500.0` (`500.0/1`), and so channelsToFlush is always `flushQ.size * 500`.

Reply:
good catch, that should be inverted.

Reply:
i had moved it to a separate val for "readability" lol.